test/functional/Valhalla/src_lw5/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -2404,28 +2381,23 @@ static public void testACMPTestOnAssortedValues() throws Throwable {
 		Object assortedValueWithLongAlignment = AssortedValueWithLongAlignment.createObjectWithDefaults();
 		Object assortedValueWithLongAlignment2 = AssortedValueWithLongAlignment.createObjectWithDefaults();
 		Object assortedValueWithLongAlignment3 = AssortedValueWithLongAlignment.createObjectWithDefaults();
-		assortedValueWithLongAlignment3 = ((AssortedValueWithLongAlignment)assortedValueWithLongAlignment3).checkFieldsWithDefaults();
+		((AssortedValueWithLongAlignment)assortedValueWithLongAlignment3).checkFieldsWithDefaults();
 
 		Object assortedValueWithObjectAlignment = AssortedValueWithObjectAlignment.createObjectWithDefaults();
 		Object assortedValueWithObjectAlignment2 = AssortedValueWithObjectAlignment.createObjectWithDefaults();
 		Object assortedValueWithObjectAlignment3 = AssortedValueWithObjectAlignment.createObjectWithDefaults();
-		assortedValueWithObjectAlignment3 = ((AssortedValueWithObjectAlignment)assortedValueWithObjectAlignment3).checkFieldsWithDefaults();
+		((AssortedValueWithObjectAlignment)assortedValueWithObjectAlignment3).checkFieldsWithDefaults();
 
-		// TODO add commented out tests once withfield is replaced
 		assertTrue((assortedValueWithLongAlignment == assortedValueWithLongAlignment), "A substitutability (==) test on the same value should always return true");
 		assertTrue((assortedValueWithObjectAlignment == assortedValueWithObjectAlignment), "A substitutability (==) test on the same value should always return true");
 		assertTrue((assortedValueWithLongAlignment == assortedValueWithLongAlignment2), "A substitutability (==) test on different value the same contents should always return true");
 		assertTrue((assortedValueWithObjectAlignment == assortedValueWithObjectAlignment2), "A substitutability (==) test on different value the same contents should always return true");
-		//assertFalse((assortedValueWithLongAlignment == assortedValueWithLongAlignment3), "A substitutability (==) test on different value should always return false");
```

Comment:
I think we should keep the test cases related to `assortedValueWithLongAlignment3` and `assortedValueWithObjectAlignment3` which have different values from the defaults.

Reply:
Okay. I've added these asserts back in and initialized assortedValueWithLongAlignment3 and assortedValueWithObjectAlignment3 with the new values that would have resulted from the old withfield tests.